### DISCUSSION TITLE
Export MXNet object detection model to ONNX

### DISCUSSION THREAD

**Discussion by ghost** [apache/incubator-mxnet#15105] [Q&A]
Hi,

I am using onnx_mxnet.export_model to export MXNet models on GluonCV model zoo to ONNX format. While majority of image classification models work well, while object detection models such as fast cnn cannot be exported due to unsupported operator in the "_contrib" family. I was wondering what is a suitable way to address this problem?

**Comment by frankfliu** [comment]
@mxnet-label-bot add [question, onnx]

**Comment by Roshrini** [comment]
Hi @tengdai722, to export object detection models, ONNX should have similar operators to which we can map MXNet contrib operators. 
There is a list of few ops mentioned here: https://github.com/apache/incubator-mxnet/issues/15181
